__tests__/isThisISOWeek_test.re

```diff
@@ -0,0 +1,57 @@
+open Jest;
+
+open Js.Date;
+
+describe("isThisISOWeek", () => {
+  open ExpectJs;
+
+  /* Source: https://blog.blakesimpson.co.uk/page.php?id=98&title=how-to-stub-date-in-jest-tests-building-a-stubdate-helper
+        and https://github.com/facebook/jest/issues/2234#issuecomment-384884729
+
+        Alternative: https://github.com/mikaello/bs-jest-date-mock
+     */
+  let stubDate: Js.Date.t => unit = [%bs.raw
```

Comment:
I am not very happy about reaching out for `bs.raw` here, but I couldn't figure out how to do this date mocking in reason. I could have installed a third party binding, e.g. [bs-jest-date-mock](https://github.com/mikaello/bs-jest-date-mock), but I tried to this without installing more dependencies.